.travis.yml

```diff
@@ -11,7 +11,10 @@ python:
 # https://github.com/travis-ci/travis-ci/issues/1147#issuecomment-441393807
 if: type != push OR branch = master OR branch =~ /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
-before_install: pip install --upgrade pip 'setuptools==65.6.2' 'poetry>=1.2.0'
+before_install: |
+  curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal
```

Comment:
What changed about 3.10 that caused it to require rust?

Reply:
But that didn't cause the build to fail? Weird.

Reply:
It did, that's why I had to fix it :) I think there might have been a recent change in one of the dependencies. What's weirder for me is that it only seems to be required for 3.10 o_O